### ISSUE TITLE
Referencing WebView2.Runtime adds dependency to Microsoft.VCLibs.140.00 and Microsoft.VCLibs.140.00.UWPDesktop

### ISSUE THREAD

**Issue by lfderm** [ProKn1fe/WebView2.Runtime#14]
When the WebView2.Runtime package is added to WinUI a project, then the following dependencies are added to the project's publish output appx package:

```
    <PackageDependency Name="Microsoft.VCLibs.140.00" MinVersion="14.0.33519.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.33728.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
```

We intended to add WebView2 Runtime to allow a self-contained deployment without any dependencies. Now we get two new dependencies added, which we would need to manually install on target machines. Newer Windows builds happen to include them already, but older ones (e.g. 22H2) do not have them and need manual installation.

Are these really needed? WinUI 3 does not need them anymore due to Hybrid CRT. They are still added by default, but they are unnecessary and can be removed by a [property switch](https://github.com/microsoft/WindowsAppSDK/discussions/3524). Can you check if they are really still needed for WebView2? Maybe they can be removed here as well?

**Comment by lfderm** [comment]
Yes, by default they still add the dependencies, although they are not used anymore. I fail to understand the logic why they are still adding the dependency. But anyways, it can be easily disabled like this:

`<WinUISDKReferences>false</WinUISDKReferences>`

If WebView2 also does not need it, it would be great if it can be removed as well, to be able to really ship a 100% self contained application.

**Comment by lfderm** [comment]
Or alternatively, if the dependencies are really required, then maybe it would be possible to just include the dlls from the corresponding appx package into the WebView2 runtime folder? The appx packages are located here (e.g. for the .Desktop x64 appx): `%ProgramFiles(x86)%\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs.Desktop\14.0\Appx\Retail\x64`

**Comment by ProKn1fe** [comment]
It seems WebView2 themself also adds Microsoft.VCLibs reference.
![image](https://github.com/user-attachments/assets/159b8c15-b6a1-4140-86de-f4c5d75f212c)

**Comment by lfderm** [comment]
I cannot confirm that. If I only reference WebView2 without WebView2.Runtime, then I don't have a dependency on it in the generated appx file. The dependency only comes if I add WebView2.Runtime.

**Comment by lfderm** [comment]
When I look into the WebView2 folder, I see multiple C Runtime dlls. So I am pretty confident that the VCLibs dependency is in fact not needed. The WebView2 Rruntime brings everything that is required. And as I said before, the WebView2 package does not have these dependencies. If I remove WebView2.Runtime and publish (using `<WinUISDKReferences>false</WinUISDKReferences>`), I have zero dependencies.
